11-Sophia-Pache/wk01/5-fri/part1/js/exercises.js

```diff
@@ -0,0 +1,131 @@
+// // "Your journey continues through the following stops: 23rd, 28th, 33rd."
+// // "7 stops in total."
+
+//     There are 3 subway lines:
+//         The N line has the following stops: Times Square, 34th, 28th, 23rd, Union Square, and 8th
+//         The L line has the following stops: 8th, 6th, Union Square, 3rd, and 1st
+//         The 6 line has the following stops: Grand Central, 33rd, 28th, 23rd, Union Square, and Astor Place.
+//         All 3 subway lines intersect at Union Square, but there are no other intersection points. (For example, this means the 28th stop on the N line is different than the 28th street stop on the 6 line, so you'll have to differentiate this when you name your stops in the arrays.)
+//     Tell the user the number of stops AND the stops IN ORDER that they will pass through or change at.
+
+// Hints:
+
+//     Work out how you would do it on paper first! Then start to explain that process in Javascript.
+//     Get the program to work for a single line before trying to tackle multiple lines.
+//     Don't worry about prompting the user for input. Hard code some values to get it working. You can use prompt() later to make it more interactive.
+//     Consider diagramming the lines by sketching out the subway lines and their stops and intersection.
+//     The key to the lab is finding the index positions of each stop. (hint: indexOf())
+//     Make sure the stops that are the same for different lines have different names (i.e. 23rd on the N and on the 6 need to be differentiated)
+
+
+    const prompt1Text = prompt("What subway line are you taking? nLine, lLine, or sixLine?");
+    const prompt2Text = prompt("What is your first stop? nLine: Time Square, 34th, 28th, 23rd, Union Square, or 8th, lLine: 4th, 3rd, Union Square, 6th, 8th, sixLine: Grand Central, 33rd, 28th, 23rd, Union Square, Astor Place.").toString();
+    const prompt3Text = prompt("What is the line of your last stop? nLine, lLine, or sixLine?");
+    const prompt4Text = prompt("What is your last stop? nLine: Time Square, 34th, 28th, 23rd, Union Square, or 8th, lLine: 4th, 3rd, Union Square, 6th, 8th, sixLine: Grand Central, 33rd, 28th, 23rd, Union Square, Astor Place.").toString();
+
+
+// Storing subway line and stops in variables.
+const nLine = ['Times Square','34th','28th','23rd','Union Square','8th'];
+const lLine = ['4th','3rd','Union Square','6th','8th'];
+const sixLine = ['Grand Central','33rd','28th','23rd','Union Square','Astor Place']
+
+
+// This function creates a new array from a loop, either normal or reverse loop, to record the stops of the trip.
+const tripLoop = function(line1, start, stop) {
+    let startIndex = line1.indexOf(start);
+    let stopIndex = line1.indexOf(stop);
+    let result = [];
+  		if (startIndex < stopIndex) {
+    		for (i=startIndex; i <= stopIndex;i++) {
+    		result.push(line1[i]);
+   			} 
+    	} else {
+			for (i=startIndex; i >= stopIndex;i--) {
+    		result.push(line1[i]);
+			}
+		}
```

Comment:
```suggestion
	if (startIndex < stopIndex) {
    		for (let i=startIndex; i <= stopIndex; i++) {
    		    result.push(line1[i]);
   		} 
    	} else {
		for (let i=startIndex; i >= stopIndex;i--) {
    		    result.push(line1[i]);
		}
	}
```
Firstly, don't forget to declare your variables with the `const` or `let` keyword!

In terms of code "cleanliness", two tips are to make sure you have correct indentation and space things out a bit. This ensures your code is in a familiar format and isn't to cramped which improves readability.